modules/KalturaSupport/components/dualScreen/cuePointsManager.js

```diff
@@ -1,247 +0,0 @@
-(function (mw, $) {
```

Comment:
We moved the cuePointsManager to a shared places (as @amirch1 suggested) to be able to use the same sourcebase both in the dual screen plugin and in the webcast poll plugin.